src/lib/sed.c

```diff
@@ -387,5 +382,5 @@ const char *sed_error_text(int sed_status)
 		return NULL;
```

Comment:
with changes you introduced I think this condition could be updated in following way:
if (sed_status < 0 || sed_status > SED_FAIL) {
    return NULL;
}
for non-existing elements between SED_AUTHORITY_LOCKED_OUT and SED_FAIL null should be returned as this elements are not defined.
when new status codes will be added this function will automatically work.

Reply:
I agree with the 1st check condition. For non-existing elements between SED_AUTHORITY_LOCKED_OUT and SED_FAIL, the array has it initialized it to NULL, so we dont have to worry about it.
Yes! any new elements in between will automatically work :)